test/e2e/e2e.go

```diff
@@ -206,7 +206,7 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 // If a "report directory" is specified, one or more JUnit test reports will be
 // generated in this directory, and cluster logs will also be saved.
 // This function is called on each Ginkgo node in parallel mode.
-func RunE2ETests(t *testing.T) {
+func RunE2ETests(t *testing.T, suite string) {
```

Comment:
agree the string is being passed at a time which is after any real customizations are added (compilation), and before the time that customizations are removed (ginkgo focus).... :)

Really it should be an arg sent along side other user provided arguments and if not provided, it should be "kubernetes test suite".

Sounds better?